docker/postgres/Dockerfile

```diff
@@ -1,11 +1,8 @@
-FROM postgres:9.5
+ARG POSTGRES_VERSION=9.6
+ARG POSTGIS_VERSION=2.5
+ARG IMAGE_VARIANT=alpine
 
-RUN apt-get update \
-    && apt-get install --no-install-recommends -y \
-        postgis postgresql-9.5-postgis-2.4 postgresql-9.5-postgis-2.4-scripts
+FROM helsinkitest/postgis:${POSTGRES_VERSION}-${POSTGIS_VERSION}-${IMAGE_VARIANT}
 
-RUN localedef -i fi_FI -c -f UTF-8 -A /usr/share/locale/locale.alias fi_FI.UTF-8
```

Comment:
there were some issues with locales, when I created the inital Dockerfiles. you sure it works now fine for Finnish?

Reply:
No, I'm not sure it works for Finnish now. I wanted to face the issue first, however, and then add those locale definitions. So far, the builds succeed, the `yso` and `tprek` importers add a lot of Finnish content to database and it works just fine, so I didn't have a reason to add those yet 🤷‍♂

Reply:
ok, it might be okay then. let's go with your version 🙂